### PR TITLE
introduction & contracts - change suggestions

### PR DESCRIPTION
**Pull request by ilinzweilin**

### CODE REVIEW

File: docs/tinlake/contracts/contracts.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ In the above diagram, all green method
...
-1) Pile: keeps track of the outstanding debt per loan
+It depends on other contracts for certain parts of the logic. These contracts can be customized to fit a deployment's needs:  
+1) Pile: keeps track of the outstanding debt per loan  
 2) Ceiling: this module can be used to control when and how much may be borrowed/repaid per loan and wh.
 
 It also allows for the collector to `seize` bad loans (take the NFT away from the borrower).
 
 #### `issue(address registry, uint token)`
-Issue a loan for the specified collateral NFT (registry address & token). Internally calls `Title` to issue a loan NFT into the sender's account.
+Issue a loan for the specified collateral NFT (registry address & token). Internally calls `Title` to issue a loan NFT into the sender's account. Only successful if sender is also the owner of the collateral NFT.
```
Comment (denniswell) [inline docs/tinlake/contracts/contracts.md]: This is only successful if the sender is also the owner of the collateral NFT.

File: docs/tinlake/contracts/contracts.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ This method does a few calls:
 * Ceiling.repay(loan, wad) is called and needs to not revert to make the Ceiling aware of the decrease in debt
 
 ### Pile
-The default implementation of the Pile allows creating of different interest rate groups and assigning each loan a rate group. Each interest rate group has an interest rate that is calculated on a per second compounding basis.
+The default implementation of the `Pile` allows creating of different interest rate groups and assigning each loan a rate group. Each interest rate group has an interest rate that is calculated on a per second compounding basis.
```
Comment (denniswell) [inline docs/tinlake/contracts/contracts.md]: ... ion of the `Pile` allows the creation of different interest rate groups and assigning each loan to a rate group...

File: docs/tinlake/contracts/contracts.md [hunk 3]
```diff
@@ -72,25 +72,30 @@ The method `accrue(uint loan)` needs to be called by the Shelf before any modifi
...
 
 ## Lender Contracts
-The Lender Contracts interact with the borrower side by supplying an amount of Currency ERC20 as requested by the Shelf.balanceRequest() method.
+The Lender Contracts interact with the borrower side by supplying an amount of Currency ERC20 as requested by the `Shelf.balanceRequest()` method.
 
 ![Lender Contracts Call Graph](../../../src/images/tinlake/lender.svg)
 
 
-Tinlake enables lenders to invest two differenct tranches: A senior tranche issuing a token called DROP, and a junior tranche issuing a token called TIN. This structure is similar to [senior/junior structures in finance](blah).
+Tinlake enables lenders to invest in two differenct tranches: A senior tranche issuing a token called DROP, and a junior tranche issuing a token called TIN. This structure is similar to [senior/junior structures in finance](blah).
```
Comment (denniswell) [inline docs/tinlake/contracts/contracts.md]: I assume, (blah_ shoud be alink to tindrop.md?)

Comment (ilinzweilin) [reply docs/tinlake/contracts/contracts.md]: This is a good question :D

Comment (denniswell) [reply docs/tinlake/contracts/contracts.md]: Pretty sure is should ;-). Let's either delete or add the link

**Pull request merged**